var/spack/repos/jcsda-emc/packages/crtm-fix/package.py

```diff
@@ -20,17 +20,20 @@ class CrtmFix(Package):
     version('2.3.0_emc', sha256='fde73bb41c3c00666ab0eb8c40895e02d36fa8d7b0896c276375214a1ddaab8f')
 
     variant('big_endian', default=True, description='Install big_endian fix files')
-    variant('little_endian', default=False, description='Install little endian fix files')
+    variant('little_endian', default=False,
+            description='Install little endian fix files')
```

Comment:
I guess the spack linter will complain about these, the usually want four spaces hanging indent (everywhere in this PR)

Reply:
I'll have to get that format setup. I started using PEP 8 in VSCode, which is what I thought Spack used.

Reply:
Would be nice if we could use directly what is coming from spack ...

Reply:
Figured out how to get Flake8 args VSCode and added these rules: https://github.com/spack/spack/blob/develop/.flake8. 

`spack style` also supports some auto fix by passing -f. We can get the code into compliance and then enable checking.